README.md

```diff
@@ -15,12 +15,10 @@
     - [Tech Stack](#tech-stack)
     - [Key Features](#key-features)
 
-  - [Queries screenshots](#queries)
 - [💻 Getting Started](#getting-started)
   - [Setup](#setup)
   - [Prerequisites](#prerequisites)
   - [Install](#install)
-  - [Usage](#usage)
   - [Run tests](#run-tests)
   - [Deployment](#triangular_flag_on_post-deployment)
```

Comment:
![image](https://github.com/ridabensalem/blog-app/assets/93039359/6e4ceeaa-aace-4384-9414-53b966c38de3)

- [ ] It would be nice if you fix the highlighted links because these are broken 😨 right now so kindly fix them or remove them to make your readme descriptive and professional 👍